sales_team_operating_unit/tests/test_crm_team_operating_unit.py

```diff
@@ -0,0 +1,66 @@
+# Copyright 2016-17 Eficent Business and IT Consulting Services S.L.
+#   (http://www.eficent.com)
+# Copyright 2017-TODAY Serpent Consulting Services Pvt. Ltd.
+#   (<http://www.serpentcs.com>)
+# License LGPL-3.0 or later (https://www.gnu.org/licenses/lgpl.html).
+from odoo.tests import common
+
+
+class TestSaleTeamOperatingUnit(common.TransactionCase):
+
+    def setUp(self):
+        super(TestSaleTeamOperatingUnit, self).setUp()
+        self.res_users_model = self.env['res.users']
```

Comment:
`.with_context(tracking_disable=True, no_reset_password=True)` to avoid mail.thread overhead and avoid email creation overhead (and they are actually sent, BTW :))

Reply:
@simahawk Most of those issues were long fixed as part of rebase.  Not sure how you even found them given the code was force pushed and overwritten.